src/Orleans/MultiCluster/IProtocolParticipant.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Threading.Tasks;
+using Orleans.Concurrency;
+
+namespace Orleans.MultiCluster
+{
+    /// <summary>
+    /// Grain interface for grains that participate in multi-cluster-protocols.
+    /// </summary>
+    public interface IProtocolParticipant  : IGrain  
```

Comment:
It is used by replicated grains (and event-sourced grains). I will move it into that PR.